fdbserver/DDTxnProcessor.actor.cpp

```diff
@@ -336,11 +341,11 @@ class DDTxnProcessorImpl {
 					std::sort(dataMove->primaryDest.begin(), dataMove->primaryDest.end());
 					std::sort(dataMove->remoteDest.begin(), dataMove->remoteDest.end());
 
-					auto ranges = result->dataMoveMap.intersectingRanges(meta.range);
+					auto ranges = result->dataMoveMap.intersectingRanges(meta.ranges.front());
```

Comment:
This seems indicate that only the first range will be used. Will there be a follow up PR to use all the ranges in the metadata?

Reply:
Yes, in the future, a dataMove could contain multiple keyranges